javaagent/src/main/kotlin/com/epam/SimpleClassTransformer.kt

```diff
@@ -0,0 +1,175 @@
+package com.epam
+
+import javassist.*
+import javassist.expr.ExprEditor
+import javassist.expr.MethodCall
+import org.slf4j.LoggerFactory
+import java.io.ByteArrayInputStream
+import java.lang.instrument.ClassFileTransformer
+import java.security.ProtectionDomain
+import java.util.concurrent.atomic.AtomicInteger
+
+class SimpleClassTransformer : ClassFileTransformer {
+
+    private val logger = LoggerFactory.getLogger(SimpleClassTransformer::class.java)
+
+    private val classCounter = AtomicInteger()
+
+    private val countProjectClasses = AtomicInteger()
+
+    override fun transform(
+        loader: ClassLoader?,
+        className: String?,
+        classBeingRedefined: Class<*>?,
+        protectionDomain: ProtectionDomain?,
+        classfileBuffer: ByteArray?
+    ): ByteArray {
+
+        val clazzName = className!!.replace("/".toRegex(), ".")
```

Comment:
as far as I can remember, using "!!" allowed when calling Java functions that definitely cannot return null, in this case, **null is impossible since there cannot be a class without a name**